bin/rhts-abort

```diff
@@ -41,7 +53,7 @@ def abortJob(server, jobid):
             resp = server.watchdog.abortJob(jobid)
             break
         except:
-            print "Unable to connect to server, sleeping 5 seconds..."
+            print ("Unable to connect to server, sleeping 5 seconds...")
```

Comment:
:eyes:

Reply:
I feel bad about the eyes emoji, my mom will receive a report about this.
She loves me.